src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar container__calendar calendar--start-day-- calendar--month-length--">
```

Comment:
set some initial values
```suggestion
    <div class="calendar container__calendar calendar--start-day--mon calendar--month-length--29">
```